modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -110,7 +224,6 @@ It now uses the common SDK format.
 |Linux    | x86_64 (musl) | PHP 8.1 | NTS |https://packages.couchbase.com/clients/php/couchbase-4.2.5-php8.1-nts-linux-musl-x86_64.tgz[couchbase-4.2.5-php8.1-nts-linux-musl-x86_64.tgz]
 |Linux    | x86_64 (musl) | PHP 8.2 | NTS |https://packages.couchbase.com/clients/php/couchbase-4.2.5-php8.2-nts-linux-musl-x86_64.tgz[couchbase-4.2.5-php8.2-nts-linux-musl-x86_64.tgz]
 |Linux    | x86_64 (musl) | PHP 8.3 | NTS |https://packages.couchbase.com/clients/php/couchbase-4.2.5-php8.3-nts-linux-musl-x86_64.tgz[couchbase-4.2.5-php8.3-nts-linux-musl-x86_64.tgz]
-|Linux    | x86_64 (musl) | PHP 8.4 | NTS |https://packages.couchbase.com/clients/php/couchbase-4.2.5-php8.4-nts-linux-musl-x86_64.tgz[couchbase-4.2.5-php8.4-nts-linux-musl-x86_64.tgz]
```

Comment:
4.2.5 did not produce binaries for musl and php 8.4, because at the time PHP 8.4 was not officially available on Alpine Linux